hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson

```diff
@@ -40,6 +40,9 @@
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
 
+  // Add exclusion files.
+  vcs_cov_excl_files: ["{proj_root}/hw/ip/hmac/dv/cov/sysrst_ctrl_cov_unr_excl.el"]
```

Comment:
Shouldn't this be 
vcs_cov_excl_files: ["{proj_root}/hw/ip/sysrst_ctrl/dv/cov/sysrst_ctrl_cov_unr_excl.el"]

Reply:
you're right, I messed up this file with hmac hjson